tests/Unit/MigratorFactoryTest.php

```diff
@@ -18,6 +18,7 @@
 use PHPCR\Migrations\VersionStorage;
 use PHPCR\SessionInterface;
 use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
```

Comment:
is this needed? i don't see this class using the trait.

Reply:
PHPUnit did remove `prophesize` integration in PHPUnit 10 and PHPUnit 9 will show the following message:

```
1) PHPCR\Migrations\Tests\Unit\MigratorFactoryTest::testFactory
PHPUnit\Framework\TestCase::prophesize() is deprecated and will be removed in PHPUnit 10. Please use the trait provided by phpspec/prophecy-phpunit.
```

Should we add it the trait know or should we stay with the deprecation?

Reply:
oh, then its a good idea to migrate to using the trait. but simply adding the `use` statement won't do that i think. we'd need to also use the trait in the class, no?

Reply:
I think no changes are needed as the Prophecy provides the `$this->prophesize` method which PHPUnit will remove in the next release.

Reply:
Oh no use statement was set for the trait. I tested to add it but it seems to able to be used with PHP 7.2: https://github.com/phpspec/prophecy-phpunit/blob/2d7a9df55f257d2cba9b1d0c0963a54960657177/composer.json#L17 so I will remove them currently until we increase PHP and PHPUnit version.

Reply:
ah, yeah both are called `use` :man_facepalming: - yep, lets drop it for now, until we move to php 7.3

Reply:
```suggestion
```